src/network/networkd-network.c

```diff
@@ -542,6 +543,17 @@ int network_load_one(Manager *manager, OrderedHashmap **networks, const char *fi
                 return -errno;
```

Comment:
In other places where we do stat for this purpose, we set the mtime to 0 on error. I think that's nice than propagating the error. We'll just maybe reload the config unnecessarily later.

Reply:
OK. Updated.